typescript/sdk/src/ism/types.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  DomainRoutingIsm,
+  StaticAggregationIsm,
+  StaticMultisigIsm,
+} from '@hyperlane-xyz/core';
+import type { types } from '@hyperlane-xyz/utils';
+
+import { ChainMap } from '../types';
+
+export type DeployedIsm =
+  | StaticMultisigIsm
+  | StaticAggregationIsm
+  | DomainRoutingIsm;
+
+export enum ModuleType {
+  UNUSED,
+  ROUTING,
+  AGGREGATION,
+  LEGACY_MULTISIG,
+  MULTISIG,
+}
```

Comment:
any way to pull this from the contracts?

Reply:
Doesn't appear to be part of the generated types